### PR TITLE
[BUG] Allow Nessie subtests to create repositories

### PR DESCRIPTION
**Pull request by arielshaqed**
Also improve error messages when repository creation fails.

### CODE REVIEW

File: nessie/system_test.go [hunk 1]
```diff
@@ -64,7 +75,10 @@ func createRepository(ctx context.Context, t *testing.T, name string, repoStorag
 		StorageNamespace: repoStorage,
 	})
 	require.NoErrorf(t, err, "failed to create repository '%s', storage '%s'", name, repoStorage)
-	require.Equal(t, http.StatusCreated, resp.StatusCode())
+	if resp.StatusCode() != http.StatusCreated {
+		t.Logf("failed to create repository '%s', storage '%s': response %s", name, repoStorage, string(resp.Body))
+		t.Fail()
+	}
```
Comment (itaiad200) [inline nessie/system_test.go:81]:
```suggestion
	require.Equalf(t, http.StatusCreated, resp.StatusCode(), "failed to create repository '%s', storage '%s': response %s", name, repoStorage, string(resp.Body))
```

**Review by itaiad200 — CHANGES_REQUESTED** [review]
I don't understand the issue (could you possibly link to one?), can a go test name include these chars to begin with?

**Review by itaiad200 — APPROVED** [review]
Thanks for opening the issue (and sorry for being a nag), approving with one minor suggestion.

File: nessie/system_test.go (hunk 1 above)
Comment (arielshaqed) [reply nessie/system_test.go]: Done, need to learn more about testify.

**Comment by arielshaqed** [comment]
Thanks; pulling once tests pass...

**Comment by arielshaqed** [comment]
Thanks!

**Pull request merged**